docs-src/typescript/generated/intrinsic-non-deterministic-logic-example.md

```diff
@@ -1,18 +1,18 @@
 ---
-id: non-deterministic-code-changes
+id: intrinsic-non-deterministic-logic-example
 title: BackgroundCheckNonDeterministic Workflow
 sidebar_label: intrinsic-non-deterministic-logic
 description: BackgroundCheckNonDeterministic Workflow is a non-deterministic Workflow as it uses a random number generator.
 ---
 
 <!-- DO NOT EDIT THIS FILE DIRECTLY.
-THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-typescript/blob/main/chapter_durable_execution/backgroundcheck_nondeterministic/src/workflow_dacx.ts. -->
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-typescript/blob/fixes2/chapter_durable_execution/backgroundcheck_nondeterministic/src/workflow_dacx.ts. -->
 
 One way to produce a non-deterministic error is to use a random number to determine whether to sleep inside the Workflow.
 
 If you run the `BackgroundCheckNonDeterministic` Workflow enough times, eventually you will see a Workflow Task failure.
```

Comment:
In Go this sentence comes after the subsequent code snippet, and I think it makes more sense to do it that way... I would also rephrase it slightly to "eventually, the way that sleep has been implemented here will produce a non-determinism error."

Reply:
due to DACX, moving this to after the code snippet is too many changes - and given the new direction... i am not sure worth the effort.

But I did update the sentence to your suggestion